salt/modules/vsphere.py

```diff
@@ -62,7 +62,7 @@ def __virtual__():
     return __virtualname__
 
 
-def get_coredump_network_config(host, username, password, protocol=None, port=None, esxi_host=None):
+def get_coredump_network_config(host, username, password, protocol=None, port=None, esxi_hosts=None):
```

Comment:
This code hasn't been released yet. I am just changing these parameters for functions that have been merged into Salt for less that a day. :)